### ISSUE TITLE
Indentation in if and case expressions

### ISSUE THREAD

**Issue by giorgian** [rubocop/rubocop#1323]
Consider this snippet:

<pre>
what = if foo
  42
else
  'doh!'
end
</pre>

I get:
C:  2:  3: Use 2 (not -5) spaces for indentation.
W:  5:  1: end at 5, 0 is not aligned with if at 1, 7

This other snippet, on the other end, is considered good by rubocop:

<pre>
what = if foo
         42
       else
         'doh!'
       end
</pre>

To me, this second sample is just crazy; also, almost no editor will accept to indent code at column 7.

**Comment by grosser** [comment]
using
```
Lint/EndAlignment:
  AlignWith: variable
```

but still getting:

```
lib/rspec-rerun/tasks.rb:31:13: C: Use 2 (not -5) spaces for indentation.
            opts
            ^^^^
lib/rspec-rerun/tasks.rb:32:11: C: Align else with if.
          else
          ^^^^
```

on 

```
opts = if opts.is_a? Hash
  opts
else
  { retry_count: opts }
end
```

any idea what I could do ?

**Comment by jonas054** [comment]
@grosser Run `rubocop --debug lib/rspec-rerun/tasks.rb` and check if you're really getting configuration from the file where you have set

```yaml
Lint/EndAlignment:
  AlignWith: variable
```
because the `ElseAlignment` and `IndentationWidth` cops both look at the `EndAlignment` configuration and should not report offenses in your case.

**Comment by grosser** [comment]
thanks, that worked ... an inherited file had 

```
Lint/EndAlignment:
  Enabled: false
```
